src/modules/avformat/producer_avformat.c

```diff
@@ -2792,6 +2830,27 @@ static int video_codec_init(producer_avformat self, int index, mlt_properties pr
     return self->video_index > -1;
 }
 
+static int pick_video_stream(producer_avformat self, int index)
```

Comment:
Suggestion to rename "index" to "absolute_index" for clarity.

Reply:
agreed, done